### ISSUE TITLE
Updating version 2.0 with related commits

### ISSUE THREAD

**Issue by ExperimentsInHonesty** [hackforla/website#120]
## Overview

It looks like there are commits after the 2.0 tag was applied.  Is there a way to fix that?

## Action Item
Discussion Can and should we update 2.0 to commit to 74778e6e which seems to be the end of the CNAME edits.  vs where it is now with 0f5c466.

**Comment by thekaveman** [comment]
Tags should be considered read-only once they are created. You never want to re-write the version to point to a different commit, you just tag/release a new version.

The decision can be a little arbitrary of when to create tags/versions.  It might be more confusing since our code and content are both in the repo. I look for more substantive features/fixes to increment the tag - I don't think we need to do that for every minor content addition/change. My `$0.02`.

EDIT: If we really wanted to tag a `2.0.1`, I think f422da2 probably makes the most sense as it was a configuration change.

**Comment by thekaveman** [comment]
Copying my comments from the duplicate #122:

>Typically the tag is created *after* all the commits have been made for the release. E.g. if there are some open PRs for the next release, they should all be merged into `gh-pages`, then a tag created on the HEAD.
> 
> Tag creation has to be done locally (and to push, you have to have write access on this repo):
>
> ```bash
> git checkout gh-pages
> git fetch && git pull
> git tag 2.0.1
> git push --tags
> ```
>
> Once the tag is pushed, you can start [drafting a new release](https://github.com/hackforla/website/releases/new) based on the tag.
